index.js

```diff
@@ -7,54 +7,76 @@ const path = require("path");
 app.use(express.static(path.join(__dirname, "public")));
 app.set("view engine", "ejs");
 
-// starting point coordinates
 const startLat = 43.839767;
 const startLng = 18.343170;
 
-// step distance
-const step = 0.01;
+const stepDistance = 100;
 
-// current point coordinates
 let currentLat = startLat;
 let currentLng = startLng;
 
-// grid of 25 points
+let prevLng = currentLng;
+
 const grid = [];
 for (let i = 0; i < 5; i++) {

```

Comment:
strpaj ovaj for loop u ```function generatePlanePath(startLat, startLon, teamName, gridSize) { /* ovdje trpas kod */ }```.

iz funkcije nek mi vrati jos i teamName na ovakav nacin:

```
[
	...,
	{ lat: 1.23, lon: 4.56, teamName: "ime tima iz argumenta" },
	{ lat: 7.89, lon: 9.87, teamName: "ime tima iz argumenta" },
	...,
]
```

Reply:
strpaj ovaj for loop u ```function generatePlanePath(startLat, startLon, teamName, gridSize) { /* ovdje trpas kod */ }```.

iz funkcije nek mi vrati jos i teamName na ovakav nacin:

```
[
	...,
	{ lat: 1.23, lon: 4.56, teamName: "ime time iz argumenta" },
	{ lat: 7.89, lon: 9.87, teamName: "ime time iz argumenta" },
	...,
]